androidApp/src/main/java/dev/johnoreilly/confetti/ConfettiViewModel.kt

```diff
@@ -23,7 +23,7 @@ class ConfettiViewModel(private val repository: ConfettiRepository): ViewModel()
 
     fun getSessionSpeakerLocation(session: SessionDetails): String {
         var text = session.speakers.joinToString(", ") { it.name }
-        text += " / ${session.room.name} / ${getLanguageInEmoji(session.language)}"
+        text += " / ${session.room?.name} / ${getLanguageInEmoji(session.language)}"
```

Comment:
Unfortunate consequence of some sessions not having a room (like lunch). Ideally, there is some UI for that